helper-scripts/send-calendar-invites.js

```diff
@@ -119,7 +117,7 @@ async function createEvents(jsonPath, supportName) {
 				calendarId: support.calendarID,
 				conferenceDataVersion: 1,
 				sendUpdates: 'all',
-				resource: eventResource,
+				requestBody: eventResource,
```

Comment:
`requestBody` appears to be the canonical form from https://googleapis.dev/nodejs/googleapis/latest/calendar/interfaces/Params$Resource$Events$Insert.html#requestBody but `resource` is usually converted to `requestBody` based on https://github.com/googleapis/nodejs-googleapis-common/blob/v3.2.2/src/apirequest.ts#L124-L148 but it's deprecated and won't work for every api call